data/autoyast_sle15/sle_micro/live_patching.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <install_updates config:type="boolean">false</install_updates>
+    <reg_code>{{SCC_REGCODE}}</reg_code>
+    <reg_server>{{SCC_URL}}</reg_server>
+    <addons config:type="list">
+      <addon>
+        <name>sle-module-live-patching</name>
+        <version>15.3</version>
+        <arch>{{ARCH}}</arch>
+        <media_url>https://dist.suse.de/install/SLP/SLE-15-SP3-Module-Live-Patching-GM/x86_64/DVD1/</media_url>
+        <reg_code>{{SCC_REGCODE}}</reg_code>
+      </addon>
+    </addons>
+  </suse_register>
+  <bootloader>
+    <global>
+      <timeout config:type="integer">-1</timeout>
+      <hiddenmenu>false</hiddenmenu>
+    </global>
+  </bootloader>
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <software>
+    <products config:type="list">
+      <product>SUSE-MicroOS</product>
+    </products>
+    <install_recommended config:type="boolean">true</install_recommended>
+    <packages config:type="list">
+      <package>grub2</package>
+    </packages>
+    <patterns config:type="list">
+      <pattern>basesystem</pattern>
+      <pattern>microos-container_runtime</pattern>
+      <pattern>microos-selinux</pattern>
```

Comment:
i just wonder if you need to install any package from `live-patching` module. like `kernel-livepatch-tools`? not sure tho.

Reply:
Yeah, I will need to add some stuff here.
This is still WIP and will change a lot.. actually I even shouldn't have create this PR yet... :)